apps/condo/domains/banking/schema/BankCostItem.test.js

```diff
@@ -0,0 +1,191 @@
+/**
+ * Generated by `createschema banking.BankCostItem 'name:Text;category:Relationship:BankCategory:SET_NULL'`
+ */
+
+const { makeLoggedInAdminClient, makeClient, UUID_RE, DATETIME_RE, expectToThrowGQLError, catchErrorFrom } = require('@open-condo/keystone/test.utils')
+
+const {
+    expectToThrowAuthenticationErrorToObj, expectToThrowAuthenticationErrorToObjects,
+    expectToThrowAccessDeniedErrorToObj,
+} = require('@open-condo/keystone/test.utils')
+
+const { makeClientWithNewRegisteredAndLoggedInUser } = require('@condo/domains/user/utils/testSchema')
+
+const { BankCostItem, createTestBankCostItem, updateTestBankCostItem } = require('@condo/domains/banking/utils/testSchema')
+const { createTestBankCategory } = require('../utils/testSchema')
```

Comment:
```suggestion
const { createTestBankCategory } = require('@condo/domains/banking/utils/testSchema')
```
Or just combine imports with 14 line